### PR TITLE
Fix the first two issues of #436

### PR DESCRIPTION
**Pull request by laubonghaudoi**
The instructions of data migration is not clear in the tutorial, as described in https://github.com/blitz-js/blitzjs.com/issues/436

The last two issues are remained to be fixed.

### CODE REVIEW

File: app/pages/docs/tutorial.mdx [hunk 1]
```diff
@@ -316,9 +317,36 @@ model Question {
...
+Your database is now in sync with your schema.
+```
+
+And then run:
+
+```sh
+blitz prisma generate
+```
+
+to update prisma client for that schema change.
```
Comment (flybayer) [inline app/pages/docs/tutorial.mdx:349]: This step is not needed because `blitz prisma migrate dev` already generates the client too

**Review by flybayer — COMMENTED** [review]
Thank you!

File: app/pages/docs/tutorial.mdx (hunk 1 above)
Comment (laubonghaudoi) [reply app/pages/docs/tutorial.mdx]: Thanks for pointing out! I have removed that part. Please check if it is ready to merge.

**Comment by flybayer** [comment]
yay!

**Pull request merged**